<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>438</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Trebuchet MS</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Sample</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.ico</normaloff>favicon.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>15</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>55</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>State</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_message">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>355</y>
      <width>381</width>
      <height>46</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send_message">
    <property name="geometry">
     <rect>
      <x>405</x>
      <y>355</y>
      <width>91</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_log">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>85</y>
      <width>481</width>
      <height>256</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_list_log">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>26</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_endpoint">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>376</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUrlCharactersOnly</set>
    </property>
    <property name="text">
     <string>wss://127.0.0.1:1609</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
